src/plone/restapi/locales/es/LC_MESSAGES/plone.restapi.po

```diff
@@ -0,0 +1,279 @@
+msgid "Suppress thumbs"
+msgstr "Suprimir miniaturas"
+
+#: plone/restapi/services/users/add.py:342
+msgid "The password passed as 'old_password' is wrong."
+msgstr "La contraseña pasada como \"Contraseña anterior\" no es válida."
+
+#: plone/restapi/services/users/add.py:307
+msgid "The reset_token is unknown/not valid."
+msgstr "El reset_token es desconocido/no válido."
+
+#: plone/restapi/configure.zcml:81
+msgid "Volto Blocks"
+msgstr "Bloques Volto"
+
+#: plone/restapi/services/users/update.py:119
+msgid "You are not authorized to perform this action"
+msgstr "No está autorizado a realizar esta acción."
+
+#: plone/restapi/services/email_send/post.py:91
+msgid "You are receiving this mail because ${sender_fullname} sent this message via the site ${portal_title}:"
+msgstr "Está recibiendo este correo porque ${sender_fullname} envió este mensaje a través del sitio ${portal_title}:"
+
+#: plone/restapi/services/users/add.py:114
+msgid "You can't send both password and sendPasswordReset."
+msgstr "No puede ingresar la contraseña y 'Enviar un correo electrónico de confirmación con un enlace para establecer la contraseña'."
+
+#: plone/restapi/services/users/add.py:322
+msgid "You can't set a password without a password reset token."
+msgstr "No puede establecer una contraseña sin un token de restablecimiento de contraseña."
+
+#: plone/restapi/services/users/update.py:125
+msgid "You can't update the properties of this user"
+msgstr "No puedes actualizar las propiedades de este usuario."
+
+#: plone/restapi/services/users/add.py:284
+msgid "You can't use 'reset_token' and 'old_password' together."
+msgstr "No puede utilizar 'Restablecer token' y 'Contraseña anterior' juntas."
+
+#: plone/restapi/services/users/add.py:109
+msgid "You have to either send a password or sendPasswordReset."
+msgstr "Debe ingresar una contraseña o 'Enviar un correo electrónico de confirmación con un enlace para establecer la contraseña'."
+
+#: plone/restapi/services/users/add.py:154
+msgid "You need AddPortalMember permission."
+msgstr "Necesita el permiso AddPortalMember."
+
+#: plone/restapi/services/users/add.py:329
+msgid "You need to be logged in as the user '${username}' to set the password."
+msgstr "Debe iniciar sesión como usuario '${username}' para establecer la contraseña."
+
+#. Default: "Missing dependency"
+#: plone/restapi/services/addons/addons.py:207
+msgid "dependency_missing"
+msgstr "Dependencia faltante"
+
+#. Default: "If selected, the navigation tree will only show the current folder and its children at all times."
+#: plone/restapi/services/contextnavigation/get.py:84
+msgid "help_current_folder_only"
+msgstr "Si se selecciona, el árbol de navegación siempre mostrará solo la carpeta actual y sus hijos."
+
+#. Default: "Whether or not to show the top, or 'root', node in the navigation tree. This is affected by the 'Start level' setting."
+#: plone/restapi/services/contextnavigation/get.py:69
+msgid "help_include_top_node"
```

Comment:
@davisagli I don't know why do we have these strings here... I think they are not shown anywhere in the API... we should review why we are passing them from gettext

Reply:
@erral Good point. It looks like it's a schema that was copied from plone.app.portlets. The schema is used to deserialize options from the request but its fields don't need translations.

Reply:
We can handle that in a separate PR, I will try to prepare it. Thanks!